src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleObject` as it is somewhat generic and does not enhance readability. Consider using a more descriptive name that reflects the variable's purpose, such as `styleMap` or `cssProperties`.